docs/settings/infrastructure-ui-settings.asciidoc

```diff
@@ -0,0 +1,39 @@
+[role="xpack"]
+[[infrastructure-ui-settings-kb]]
+=== Infrastructure UI Settings in Kibana
+++++
+<titleabbrev>Infrastructure UI Settings</titleabbrev>
+++++
+
+You do not need to configure any settings to use Infrastructure UI. It is enabled by default.
+
+[float]
+[[general-infra-ui-settings-kb]]
+==== General Infrastructure UI Settings
+
+xpack.infra.enabled:: Set to `false` to disabled the Logs and Infrastructure UI plugin {kib}. Defaults to
+`true`.
+
+xpack.infra.sources.default.logAlias:: Index pattern for matching indices that contain log data. Defaults to
+`filebeat-*`.
```

Comment:
Do you think it's worth showing here that you can specify more than one index pattern? I was told that you can do this (haven't tested it myself, though)

`xpack.infra.sources.default.logAlias: "filebeat-*,journalbeat-*"`